javascript/javascript1/week2/Student manager.js

```diff
@@ -0,0 +1,44 @@
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if  (studentName === "Dronning"){
+    class07Students.push(studentName);
+    }
+    else if (class07Students.length>=6) {
+     console.log("Cannot add more students to class 07");
+    }
+     else if (class07Students.includes(studentName) === true) {
+        console.log('You are allready in the class');
+    }
+      
+    else {
+ class07Students.push(studentName);
+}
+}
+function getNumberOfStudents(){
+    return class07Students.length;
+}
+    addStudentToClass('Basma');
+addStudentToClass('Basma');
+    addStudentToClass('Omar');
+    addStudentToClass('Rem'); 
+    addStudentToClass('maria');
+    addStudentToClass('Asia');
+    addStudentToClass('Aya');
+    addStudentToClass('loucas');
+    
+    addStudentToClass('Dronning');
+    
+    console.log (class07Students);
+  console.log (getNumberOfStudents());
```

Comment:
Super solution here, well done! ✨

Good use of includes(), and great reflection of the chronology of the conditions in the if else statements. 

What happens if you add "Dronning" more than once? 😊